README.md

```diff
@@ -2,391 +1,310 @@
+* The call to `LearnosityItems.init()`, which initiates Items API to inject the assessment player into the page.
+* The line `<%= app.initOptions(request.getServerName()) %>` dynamically sends the contents of our init options to JavaScript, so it can be passed to `init()`.
 
-dataApi.setRequestConfig(requestConfig);
+The call to `init()` returns an instance of the ItemsApp, which we can use to programmatically drive the assessment using its methods. We pull in our Learnosity configuration in the line `<%= app.initOptions(request.getServerName()) %>`. 
 
+There is some additional code in `index.jsp`.
 
-JSONObject response = dataApi.requestJSONObject();
-JSONObject res = new JSONObject(response.getString("body"));
+Our web app is configured to start in this block in `web.xml`:
 
-// You can also get a Remote object like this:
-Remote remote = dataApi.request();
-String body = remote.getBody();
+``` XML
+<welcome-file-list>
+    <welcome-file>index.jsp</welcome-file>
+</welcome-file-list>
 ```
 
-## Troubleshooting
+This marks the end of the quick start guide. From here, try modifying the example files yourself, you are welcome to use this code as a basis for your own projects.
 
-We recommend that you use the Java JSON implementation which comes packaged in the sdk. In the past, we had several issues which were due to incompatible JSON implementations.
+Take a look at some more in-depth options and tutorials on using Learnosity assessment functionality below.
 
+[(Back to top)](#table-of-contents)
 
-## Creating jar files (using Eclipse Luna Release 4.4.0)
+## Next steps: additional documentation
 
-The 'light' jar:
+### **SDK reference**
+See a more detailed breakdown of all the SDK features, and examples of how to use more advanced or specialised features on the [SDK reference page](REFERENCE.md).
 
-In eclipse right click the java sdk project and select 'Export'. Under 'Java', select 'JAR file'. In the following screen only select the packages in the src (excluding the library folder) and click 'Finish'.
+### **Additional quick start guides**
+There are more quick start guides, going beyond the initial quick start topic of loading an assessment, these further tutorials show how to set up authoring and analytics:
+* [Authoring Items quick start guide](https://help.learnosity.com/hc/en-us/articles/360000754958-Getting-Started-With-the-Author-API) (Author API) - create and edit new Questions and Items for your Item bank, then group your assessment Items into Activities, and
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion

* [Authoring Items quick start guide](https://help.learnosity.com/hc/en-us/articles/360000754958-Getting-Started-With-the-Author-API) (Author API) - create and edit new Questions and Items for your Item bank, then group your assessment Items into Activities, and
```